src/ad-hoc-visualizations/issues/visualization.tsx

```diff
@@ -56,10 +56,26 @@ export const IssuesAdHocVisualization: VisualizationConfiguration = {
         }),
     getIdentifier: () => AdHocTestkeys.Issues,
     visualizationInstanceProcessor: () => VisualizationInstanceProcessor.nullProcessor,
-    getNotificationMessage: selectorMap =>
-        isEmpty(selectorMap)
-            ? 'Congratulations!\n\nAutomated checks found no issues on this page.'
-            : null,
+    getNotificationMessage: (selectorMap, key, warnings) => {
```

Comment:
Not a necessity but I'd strongly like to move this out of here and have it as a floating function; I don't want to give the impression that we should be writing significant code (which I'd say this is) in the configuration files, if that makes sense.

Reply:
I was on the fence about this... right now this is the only place where we have a "complex" function to create the notification message, but I also knew we're going to need to update this as soon as we want to remove the old fast pass automated check view.